File: ALODWebUtility.CS/Common/childunit.cs
namespace ALODWebUtility.Common
{
    public class childunit
    {
        private string _CHAIN_TYPE = string.Empty;
        private string _childName = string.Empty;
        private string _childPasCode = string.Empty;
        private int _cs_id = -1;
        private int _Level = -1;
        private int _parentCS_ID = -1;
        private int _userUnit = -1;

        private bool in_Active = false;

        public childunit()
        {
        }

        public string CHAIN_TYPE
        {
            get
            {
                return this._CHAIN_TYPE;
            }
            set
            {
                this._CHAIN_TYPE = value;
            }
        }

        public string childName
        {
            get
            {
                return this._childName;
            }
            set
            {
                this._childName = value;
            }
        }

        public string childPasCode
        {
            get
            {
                return this._childPasCode;
            }
            set
            {
                this._childPasCode = value;
            }
        }

        public int cs_id
        {
            get
            {
                return this._cs_id;
            }
            set
            {
                this._cs_id = value;
            }
        }

        public bool InActive
        {
            get
            {
                return this.in_Active;
            }
            set
            {
                this.in_Active = value;
            }
        }

        public int Level
        {
            get
            {
                return this._Level;
            }
            set
            {
                this._Level = value;
            }
        }

        public int parentCS_ID
        {
            get
            {
                return this._parentCS_ID;
            }
            set
            {
                this._parentCS_ID = value;
            }
        }

        public int userUnit
        {
            get
            {
                return this._userUnit;
            }
            set
            {
                this._userUnit = value;
            }
        }
    }
}

File: ALODWebUtility/Common/childunit.vb
Namespace Common

    Public Class childunit

        Private _CHAIN_TYPE As String = String.Empty
        Private _childName As String = String.Empty
        Private _childPasCode As String = String.Empty
        Private _cs_id As Integer = -1
        Private _Level As Integer = -1
        Private _parentCS_ID As Integer = -1
        Private _userUnit As Integer = -1

        Private in_Active As Boolean = False

        Public Sub New()

        End Sub

        Public Property CHAIN_TYPE() As String
            Get
                Return Me._CHAIN_TYPE
            End Get
            Set(ByVal value As String)

                Me._CHAIN_TYPE = value

            End Set
        End Property

        Public Property childName() As String
            Get
                Return Me._childName
            End Get
            Set(ByVal value As String)

                Me._childName = value

            End Set
        End Property

        Public Property childPasCode() As String
            Get
                Return Me._childPasCode
            End Get
            Set(ByVal value As String)

                Me._childPasCode = value

            End Set
        End Property

        Public Property cs_id() As Integer
            Get
                Return Me._cs_id
            End Get
            Set(ByVal value As Integer)

                Me._cs_id = value

            End Set
        End Property

        Public Property InActive() As Boolean
            Get
                Return Me.in_Active
            End Get
            Set(ByVal value As Boolean)

                Me.in_Active = value

            End Set
        End Property

        Public Property Level() As Integer
            Get
                Return Me._Level
            End Get
            Set(ByVal value As Integer)

                Me._Level = value

            End Set
        End Property

        Public Property parentCS_ID() As Integer
            Get
                Return Me._parentCS_ID
            End Get
            Set(ByVal value As Integer)

                Me._parentCS_ID = value

            End Set
        End Property

        Public Property userUnit() As Integer
            Get
                Return Me._userUnit
            End Get
            Set(ByVal value As Integer)

                Me._userUnit = value

            End Set
        End Property

    End Class

End Namespace